prisma/seeds/seed-with-sense.ts

```diff
@@ -0,0 +1,68 @@
+import { db } from "@/server/db";
```

Comment:
we already have a seeding script, right ? How is this one different?

Reply:
Suppose you run docker-compose build for first time...where db is not seeded. Then, it checks the user count in db if count < 0 , our db needs to be seeded .
And again when you run docker-compose up for second time, it automatically checks count and skip the seeding process. 
Definitely we may reuse the previous seeding logic  but I assume we still need to implement logic whether to seed or not in some shell scripts.

I am not sure my approach is right but this may help to setup dev environment little more comfortably for subsequent times without passing some flags to seed or not each time.

Reply:
@G3root